server/views/pages/bookAVisit/dateAndTime.njk

```diff
@@ -54,6 +55,16 @@
     <div class="govuk-grid-column-full">
       {% include "partials/errorSummary.njk" %}
 
+      {% if validationMessage !== '' %}
+        {{ mojAlert({
+          variant: "error",
+          title: validationMessage,
+          showTitleAsHeading: true,
+          dismissible: false,
+          html: '<p data-test="validation-text">Select a new visit time.</p>'
```

Comment:
Doesn't look like this `data-test` is actually used in any tests? (Or in the version of this in `overbooking.njk`) Looking at the documentation, it should be possible to set `attributes: { "data-test": "validation-error" } on the component. That way you can use the `text` instead of the `html` property too so you don't need add the HTML markup.

Reply:
This data-test was used for initial implementation then I realised as the information given for that field is the same on a particular page it made no sense being so specific, I've reduced back to the `text` option